iroh-base/src/relay_url.rs

```diff
@@ -36,15 +35,20 @@ impl From<Url> for RelayUrl {
     }
 }
 
+/// Can occur when parsing a string into a [`RelayUrl`].
+#[derive(Debug, thiserror::Error)]
+#[error("Failed to parse: {0}")]
+pub struct RelayUrlParsingError(#[from] url::ParseError);
```

Comment:
I realise this is bikeshedding, but it's somewhat surprising that this is "Pars**ing**", rather than `RelayUrlParseError`

Reply:
I don't care at all 🤷

Reply:
changed